mainnet/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+services:
+  itchysats-mainnet:
+    image: ghcr.io/itchysats/itchysats/taker:0.4.19
```

Comment:
It's a pity we have to set this manually, but I guess there is not way around. One more step for releasing it annoying though. Maybe we can add automation to the release action at some point to create a PR on this repo :)
Not to be actioned now ;)

Reply:
I would love to set it to `latest` tag, when we have this feature: https://github.com/itchysats/itchysats/issues/1959